docs/scripts/live-preview.sh

```diff
@@ -44,6 +46,12 @@ bazel build //templates:create-daml-app-docs
 mkdir -p $BUILD_DIR/source/getting-started/code/templates-tarball
 tar -zxf ../../bazel-bin/templates/create-daml-app-docs.tar.gz -C $BUILD_DIR/source/getting-started/code/templates-tarball/
 
+# Templates
```

Comment:
It seems more logical to have the templates-tarball produced by the templates/BUILD file based on files in templates, and then have the docs depend on that, than have the docs depend on inline files and then have the templates/BUILD file grab source code from the docs.

Reply:
Also, yes, because it's a patchfile, something somewhere has to know how to produce the complete template, and it seems better for that to be templates than docs. Not having it as a patchfile seemed way worse, as we definitely want it in sync with the create-daml-app template.